proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -29,9 +29,9 @@ enum BreakingChangeCategory {
   BREAKING_CHANGE_CATEGORY_WIRE_JSON = 2;
 }
 
-message BreakingChangeSettings {
-  bool enabled = 1;
-  bool ignore_unstable_packages = 2;
+message BreakingChangePolicy {
+  optional bool enabled = 1;
```

Comment:
Why was `optional` added here?

Reply:
For partial updates, so we could differentiate between set vs unset bool. The alternative was to use BoolValue, but historically we abused `optional` for this purpose.